apps/web/src/views/BuyCrypto/containers/BuyCryptoForm.tsx

```diff
@@ -112,47 +117,53 @@ export function BuyCryptoForm({
 
   return (
     <>
-      <FormHeader title={t('Buy Crypto')} subTitle={t('Buy crypto in just a few clicks')} />
-      <FormContainer>
-        <Box>
-          <CurrencyInputPanel
-            hideBalanceComp
-            id="onramp-input"
-            showMaxButton={false}
-            value={typedValue}
-            currency={outputCurrency}
-            onUserInput={handleTypeOutput}
-            onCurrencySelect={handleOutputSelect}
-            error={Boolean(error)}
-            showCommonBases={false}
-            tokensToShow={fiatCurrencyMap as any}
-            title={
-              <Text px="4px" bold fontSize="12px" textTransform="uppercase" color="secondary">
-                {t('I want to spend')}
-              </Text>
-            }
-          />
-          {error ? (
-            <Text pb="16px" fontSize="12px" color="red">
-              {error}
-            </Text>
-          ) : null}
-        </Box>
-        <CenterWrapper>
-          <ArrowDownIcon className="icon-down" color="primary" width="22px" />
-        </CenterWrapper>
-
-        <AssetSelect onCurrencySelect={handleInputSelect} currency={inputCurrency} />
-        <Text color="textSubtle" fontSize="14px" px="4px">
-          {t('Proceed to get live aggregated quotes from a variety of different fiat onramp providers.')}
-        </Text>
-        <GetQuotesButton
-          errorText={inputError}
-          modalView={modalView}
-          setModalView={setModalView}
-          fetchQuotes={fetchQuotes}
-        />
-      </FormContainer>
+      <div style={{ padding: '4px' }}>
```

Comment:
a tip - you could always use a `Box` component

Reply:
And no need for the segment component wrapper `<></>` if there's only one react node returned

Reply:
reolved yes use box here now